modules/core/src/utils/webvr.js

```diff
@@ -0,0 +1,31 @@
+/* global document */
```

Comment:
Quite a bit of additional lines in AnimationLoop, wonder if some of the WebVR stuff could go here.

Also let's think about tree-shaking, many apps don't want to pay for AnimationLoop if they don't use it. So put that code in a class and let AnimationLoop take that class as an argument?

Reply:
Very much agree with this. Add some support hooks in animation loop to render more than once, but keep webvr specific logic out of the core render loop.

Reply:
Moving to VRAnimationLoop should solve all these.